src/toolkits/drawing_classifier/drawing_classifier.cpp

```diff
@@ -234,11 +234,11 @@ void drawing_classifier::init_options(
       FLEX_UNDEFINED,
       std::numeric_limits<int>::min(), std::numeric_limits<int>::max());
 
-  options.create_string_option(
+  options.create_boolean_option(
       "warm_start",
       "Record warm start model version used. If no warmstart used"
-      " None is assigned by default."
-      "",
+      " None is assigned by default.",
```

Comment:
I'm not sure I'm doing the right thing. do @fareeha you want to store the warm start model name at here?

Reply:
@Jarvi-Izana let's revert this change. I think we want this to be a string option but I'll let @fareeha confirm.

Reply:
yeah, modified. please have a look!